core/templates/pages/skill-editor-page/skill-preview-tab/skill-preview-tab.component.ts

```diff
@@ -54,10 +54,11 @@ angular.module('oppia').component('skillPreviewTab', {
       var ctrl = this;
 
       ctrl.$onInit = function() {
+        ctrl.INTERATION_TYPE_ALL = 'All';
```

Comment:
If this is not used in the template, do not declare it on ctrl. Make it a local variable instead.

Reply:
Done